e2e/capl-cluster-flavors/kubeadm-full-capl-cluster/chainsaw-test.yaml

```diff
@@ -0,0 +1,249 @@
+            namespace: ($namespace)
+
+    # Get the KUBECONFIG of the child cluster for later use
+    - name: Get child cluster kubeconfig
+      try:
+        - script:
+            env:
+              - name: CLUSTER
+                value: ($cluster)
+              - name: NAMESPACE
+                value: ($namespace)
+              - name: CLUSTERCTL_CONFIG
+                value: (env('CLUSTERCTL_CONFIG'))
+            content: |
+              set -e
+              clusterctl get kubeconfig $CLUSTER -n $NAMESPACE > kubeadm-full-cluster-kubeconfig.yaml
+            check:
+              ($error == null): true
+
+    # Test for child cluster resources
+    - clusters:
+        kubeadm-full-cluster:
+          kubeconfig: ./kubeadm-full-cluster-kubeconfig.yaml
+      name: Check child cluster resources
+      try:
+        - assert:
+            cluster: kubeadm-full-cluster
+            file: assert-child-cluster-deployments.yaml
+        - assert:
+            cluster: kubeadm-full-cluster
+            file: assert-child-cluster-daemonsets.yaml
+      catch:
+        - describe:
+            cluster: kubeadm-full-cluster
+            apiVersion: apps/v1
+            kind: Deployment
+            namespace: kube-system
+        - describe:
+            cluster: kubeadm-full-cluster
+            apiVersion: apps/v1
+            kind: DaemonSet
+            namespace: kube-system
+
+    # Test available StatefulSets
+    - clusters:
+        kubeadm-full-cluster:
+          kubeconfig: ./kubeadm-full-cluster-kubeconfig.yaml
+      name: Check the available StatefulSets
+      try:
+        - assert:
+            cluster: kubeadm-full-cluster
+            file: assert-child-cluster-statefulsets.yaml
+      catch:
+        - describe:
+            cluster: kubeadm-full-cluster
+            apiVersion: apps/v1
+            kind: StatefulSet
+            namespace: kube-system
+
+    # Test Konnectivity server and agent
+    - clusters:
+        kubeadm-full-cluster:
+          kubeconfig: ./kubeadm-full-cluster-kubeconfig.yaml
+      name: Check konnectivity resources
+      try:
+        - assert:
+            cluster: kubeadm-full-cluster
+            file: assert-konnectivity-resources.yaml
+      catch:
+        - describe:
+            cluster: kubeadm-full-cluster
+            apiVersion: apps/v1
+            kind: Deployment
+            namespace: kube-system
+        - describe:
+            cluster: kubeadm-full-cluster
+            apiVersion: apps/v1
+            kind: DaemonSet
+            namespace: kube-system
+
+    # Test to check if child cluster is deleted
+    - name: Delete child cluster
+      try:
+        - delete:
+            ref:
+              apiVersion: cluster.x-k8s.io/v1beta1
+              kind: Cluster
+              name: ($cluster)
+        - delete:
+            ref:
+              apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+              kind: LinodeVPC
+              name: ($cluster)
```

Comment:
question: Does deleting the `LinodeVPC` resource clean up the VPC on linode cloud manager?

Reply:
Yup